lineage/figures/figure6.py

```diff
@@ -1,665 +1,16 @@
-    ax[i].set_xlim((0, int(np.ceil(1.1 * max(x_unpruned)))))
-    ax[i].set_xlabel('Number of Cells', fontsize=font2)
-    ax[i].scatter(x_pruned, res[0], c='b', marker="o", label='Susceptible Pruned', alpha=0.5)
-    ax[i].scatter(x_pruned, res[1], c='r', marker="o", label='Resistant Pruned', alpha=0.5)
-    ax[i].set_ylabel(r'Gamma a $\beta$', rotation=90, fontsize=font2)
-    ax[i].axhline(y=gamma_a0, linestyle=(0, (3, 5, 1, 5, 1, 5)), linewidth=2, color='b', alpha=0.6)
-    ax[i].axhline(y=gamma_a1, linestyle=(0, (3, 5, 1, 5, 1, 5)), linewidth=2, color='r', alpha=0.6)
-    ax[i].set_title('Gamma', fontsize=font)
-    ax[i].tick_params(axis='both', which='major', labelsize=10, grid_alpha=0.25)
-    ax[i].legend(loc='best', framealpha=0.3)
-
-    i += 1
-    res = [[i for i, j in gamma_scale_pruned], [j for i, j in gamma_scale_pruned]]
-    ax[i].set_xlim((0, int(np.ceil(1.1 * max(x_unpruned)))))
-    ax[i].set_xlabel('Number of Cells', fontsize=font2)
-    ax[i].scatter(x_pruned, res[0], c='b', marker="o", label='Susceptible Pruned', alpha=0.5)
-    ax[i].scatter(x_pruned, res[1], c='r', marker="o", label='Resistant Pruned', alpha=0.5)
-    ax[i].set_ylabel(r'Gamma scale $\alpha$', rotation=90, fontsize=font2)
-    ax[i].axhline(y=gamma_scale0, linestyle=(0, (3, 5, 1, 5, 1, 5)), linewidth=2, color='b', alpha=0.6)
-    ax[i].axhline(y=gamma_scale1, linestyle=(0, (3, 5, 1, 5, 1, 5)), linewidth=2, color='r', alpha=0.6)
-    ax[i].set_title('Gamma', fontsize=font)
-    ax[i].tick_params(axis='both', which='major', labelsize=10, grid_alpha=0.25)
-    ax[i].legend(loc='best', framealpha=0.3)
-
-    i += 1
-    ax[i].set_xlim((0, int(np.ceil(1.1 * max(x_pruned)))))
-    ax[i].set_xlabel('Number of Cells', fontsize=font2)
-    ax[i].scatter(x_pruned, tr_prunedNorm, c='k', marker="o", label=' Pruned', alpha=0.5)
-    ax[i].set_ylabel(r'$||T-T_{est}||_{F}$', rotation=90, fontsize=font2)
-    ax[i].axhline(y=0, linestyle=(0, (3, 5, 1, 5, 1, 5)), linewidth=2, color='k', alpha=0.6)
-    ax[i].set_title('Norm Transition', fontsize=font)
-    ax[i].tick_params(axis='both', which='major', labelsize=10, grid_alpha=0.25)
-    ax[i].legend(loc='best', framealpha=0.3)
-
-    i += 1
-    ax[i].set_xlim((0, int(np.ceil(1.1 * max(x_pruned)))))
-    ax[i].set_xlabel('Number of Cells', fontsize=font2)
-    ax[i].scatter(x_pruned, pi_prunedNorm, c='k', marker="o", label=' Pruned', alpha=0.5)
-    ax[i].set_ylabel(r'$||\pi-\pi_{est}||_{2}$', rotation=90, fontsize=font2)
-    ax[i].axhline(y=0, linestyle=(0, (3, 5, 1, 5, 1, 5)), linewidth=2, color='k', alpha=0.6)
-    ax[i].set_title('Norm Pi', fontsize=font)
-    ax[i].tick_params(axis='both', which='major', labelsize=10, grid_alpha=0.25)
-    ax[i].legend(loc='best', framealpha=0.3)
-
-
-# ----------- Figure 6 for G1G2
-
-def accuracy_increased_cellsG():
-    """ Calclates accuracy and parameter estimation by increasing the number of cells in a lineage for a two-state model. """
-
```

Comment:
You're sure you want to delete all this?

Reply:
Oh so all the figure changes, @Farnazmdi and I made on another branch. There's about 8(ish) new figures on there, waiting to be merged. I'm deleting all the figure stuff on this branch for that reason (since, they're all primed and ready to go in another branch apart from some minor tweaks).

Reply:
Ok great.